dcat/index.html

```diff
@@ -1872,6 +1876,83 @@ <h4>Property: was generated by</h4>
         </section>
 
     </section> <!-- end class Dataset -->
+<section id="Class:Dataset_Series">
+    
+        <h3>Class: Dataset Series</h3>
+
+        <aside class="note">
+            <p>
+              Class added in DCAT 3, see <a href="https://github.com/w3c/dxwg/issues/1272">Issue #1272</a>.
+            </p>
+            
+        </aside>
+        
+        <p>The following property is specific to this class:
+            <a href="#Property:dataset_series_has_part">hasPart</a>.
+            
+        </p>
+        <p>The following properties of the super-class <a href="#Class:Resource"><code>dcat:Resource</code></a> and <a href="#Class:Dataset"><code>dcat:Dataset</code></a>  are also available for use:
```

Comment:
--> "The following properties of the super-classes" (plural)

Reply:
thanks, corrected